backend/src/bundles/lms-data/lms-data.ts

```diff
@@ -0,0 +1,11 @@
+import { LMSDataModel } from './lms-data.model.js';
+import { LMSDataRepository } from './lms-data.repository.js';
+import { LMSDataService } from './lms-data.service.js';
+
+const lmsDataRepository = new LMSDataRepository(LMSDataModel);
+const lmsDataService = new LMSDataService(
+    lmsDataRepository,
+    process.env.X_TOKEN,
```

Comment:
We have config package. 
I think it's better to use package.
```suggestion
    config,
```